source/less/global/fonts.less

```
@@ -0,0 +1,15 @@
+@font-face {
+	font-family: "PT Sans";
+	src: url("../fonts/ptsans.woff2") format("woff2"), url("../fonts/ptsans.woff") format("woff");
+	font-weight: 400;
+	font-style: normal;
+	font-display: swap;
+}
+
+@font-face {
+	font-family: "PT Sans";
+	src: url("../fonts/ptsansbold.woff2") format("woff2"), url("../fonts/ptsansbold.woff") format("woff");
+	font-weight: 700;
+	font-style: bold;
```

```suggestion
	font-style: normal;
```
Действительные значения для этого свойства: https://webref.ru/css/font-style